src/main/java/org/opensearch/security/support/SecurityUtils.java

```diff
@@ -96,16 +96,16 @@ public static String replaceEnvVars(String in, Settings settings) {
             return in;
         }
 
-        return replaceEnvVarsBC(replaceEnvVarsNonBC(replaceEnvVarsBase64(in)));
+        return replaceEnvVarsBC(replaceEnvVarsNonBC(replaceEnvVarsBase64(in, settings), settings), settings);
     }
 
-    private static String replaceEnvVarsNonBC(String in) {
+    private static String replaceEnvVarsNonBC(String in, Settings settings) {
```

Comment:
@terryquigleysas Do you think its worth it at all to introduce a more generic `envhash`, while maintaining support for `envbc` (See prefixes [here](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/support/SecurityUtils.java#L48)). `bc` will become a legacy term for any clusters switching hashing strategies. There is still legacy terms in this repo (i.e. `.opendistro_security` index and references to `kibana`) so its not unprecedented.

Reply:
@terryquigleysas Yes, I think that would makes sense for the future.

Reply:
Marking this thread as resolve and merging this PR. Thanks for the contribution @terryquigleysas!